api_tests/tokens/views/test_token_detail.py

```diff
@@ -126,499 +165,354 @@ def test_updating_one_field_should_not_blank_others_on_patch_update(
         assert res.status_code == 200
         assert 'token_id' not in res.json['data']['attributes']
 
     @mock.patch('framework.auth.cas.CasClient.revoke_tokens')
     def test_update_token_does_not_return_token_id(
-            self, mock_revoke, app, url_token_detail, user_one):
-        mock_revoke.return_value = True
-        correct = post_payload()
+            self,
+            mock_method,
+            app,
+            url_token_detail,
+            user_one,
+            write_scope
+    ):
+        mock_method.return_value(True)
         res = app.put_json_api(
             url_token_detail,
-            correct,
+            self.post_attributes_payload(scopes='osf.full_write'),
             auth=user_one.auth,
-            expect_errors=True)
+            expect_errors=True
+        )
         assert res.status_code == 200
         assert 'token_id' not in res.json['data']['attributes']
 
     @mock.patch('framework.auth.cas.CasClient.revoke_tokens')
-    def test_update_token(self, mock_revoke, app, user_one, url_token_detail):
-        mock_revoke.return_value = True
-        correct = post_payload()
-        res = app.put_json_api(
+    def test_update_token(
+            self,
+            mock_method,
+            app,
+            user_one,
+            token_user_one,
             url_token_detail,
-            correct,
-            auth=user_one.auth,
-            expect_errors=True)
-        assert res.status_code == 200
-
-    @mock.patch('framework.auth.cas.CasClient.revoke_tokens')
-    def test_update_token_add_scope(self, mock_revoke, app, user_one, token_user_one, url_token_detail):
-        mock_revoke.return_value = True
-        original_scope = token_user_one.scopes.first()
-        scope = ApiOAuth2ScopeFactory()
-        correct = post_payload(scopes=scope.name)
+            write_scope,
+    ):
+        mock_method.return_value(True)
         res = app.put_json_api(
             url_token_detail,
-            correct,
+            self.post_attributes_payload(scopes=write_scope.name),
             auth=user_one.auth,
-            expect_errors=True)
+            expect_errors=True
+        )
         assert res.status_code == 200
-        scopes_data = res.json['data']['embeds']['scopes']['data']
-        assert len(scopes_data) == 1
-        assert scopes_data[0]['id'] == scope.name
-        assert scope.name != original_scope.name
+        assert res.json['data']['relationships']['scopes']
+        token_user_one.reload()
+        assert token_user_one.scopes.all().first().name == write_scope.name
 
-    @pytest.mark.enable_implicit_clean
     @mock.patch('framework.auth.cas.CasClient.revoke_tokens')
-    def test_token_detail_crud_with_wrong_payload(
-            self, mock_revoke, app, url_token_list, url_token_detail,
-            token_user_one, user_one, user_two):
-        mock_revoke.return_value = True
-
-        # test_non_owner_cant_delete
```

Comment:
These were made single test cases and moved to `api_tests/tokens/views/test_token_detail_relationships.py`